assets/src/edit-story/components/library/test/mediaElement.js

```diff
@@ -0,0 +1,125 @@
+/*
+ * Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * External dependencies
+ */
+import { FlagsProvider } from 'flagged';
+
+/**
+ * Internal dependencies
+ */
+import { Simulate } from 'react-dom/test-utils';
+import MediaElement from '../panes/media/common/mediaElement';
+import { ProviderType } from '../panes/media/common/providerType';
+import { renderWithTheme } from '../../../testUtils';
+
+function renderMediaElement(resource, providerType) {
+  const { getByAriaLabel, queryByAriaLabel } = renderWithTheme(
```

Comment:
Why not just return the result of renderWithTheme?

Reply:
Done.